.circleci/config.yml

```diff
@@ -191,8 +209,13 @@ workflows:
           filters:
             branches:
               ignore: [main, gh-pages]
-      - "Snyk scan":
-          context: ["snyk-apps"]
+      - snyk-deps:
+          context: ["tiki-snyk"]
```

Comment:
@tommyknows do you remember why we used this other context before? As it stands now I've made it similar to kubernetes-resource-store.

Reply:
I probably copy-pasted this from somewhere else. IIRC in container we've used that snyk-apps context everywhere (not sure if we still do though). 

I didn't know that we had a separate context for it, TIL.